.gitignore

```diff
@@ -39,3 +39,6 @@ labels.txt
 *.chi
 *.CSV
 *.thm
+
+#IDE Folders
+.vscode/
```

Comment:
It is a personal opinion; please feel free to ignore it. I think we should try to have a gitignore that only houses package-specific things.